components/bl-dashlet/src/index.js

```diff
@@ -46,7 +46,7 @@ export default function DashletComponent({ component, eventHandlers, pods, insta
   return (
     <div
       ref={ rootRef }
-      className={ cn('bl-customComponent-dashlet', classList) }
+      className={ cn('bl-customComponent-dashlet', classList, { dragging }) }
```

Comment:
Are use sure that is `dragging`? Dragging - is a process, if we want to say: dashlet could be dragged - there should be `draggable`